rel/apps/couch_epi.config

```diff
@@ -0,0 +1,9 @@
+{plugins, [
+    couch_db_epi,
+    chttpd_epi,
+    couch_index_epi,
+    global_changes_epi,
+    mango_epi,
+    mem3_epi,
+    setup_epi
```

Comment:
We shouldn't have this in rel/files/sys.config. I reverted  [commit](https://github.com/cloudant/couchdb/commit/ff53ff84d6fce4dc285797deb4d8840e3d2f1e40)